src/components/useraccount/SidebarMenu.vue

```diff
@@ -1,14 +1,26 @@
 <template>
   <div class="my-account-sidebar">
     <div id="my-account-mobile-content"
-          class="my-account-sidebar-items active">
-      <span>Personal Details</span>
+          class="my-account-sidebar-items"
+          :class="{ 'active': showTab.showPersonalDetails }">
```

Comment:
active should not be wrapped in single quotes as you can I could see int the Vuejs documentation. 
source: https://vuejs.org/v2/guide/class-and-style.html#ad